central/debug/service/singleton.go

```diff
@@ -30,7 +31,8 @@ func initialize() {
 		groupDataStore.Singleton(),
 		roleDataStore.Singleton(),
 		configDS.Singleton(),
-		notifierDS.Singleton())
+		notifierDS.Singleton(),
+		deleRegDS.Singleton())
```

Comment:
nit: just in case we need new field to have 1 line diff
```suggestion
		deleRegDS.Singleton(),
		)
```